conanfile.py

```diff
@@ -95,41 +100,42 @@ def build_autotools(self):
                                    'RANLIB=:'])
             env_vars['win32_target'] = '_WIN32_WINNT_VISTA'
 
-            with tools.chdir(self.archive_name):
+            with tools.chdir(self._source_subfolder):
                 tools.run_in_windows_bash(self, 'chmod +x build-aux/ar-lib build-aux/compile')
 
         if rc:
             configure_args.extend(['RC=%s' % rc, 'WINDRES=%s' % rc])
 
-        with tools.chdir(self.archive_name):
+        with tools.chdir(self._source_subfolder):
             with tools.environment_append(env_vars):
                 env_build.configure(args=configure_args, host=host, build=build)
                 env_build.make()
-                env_build.make(args=["install"])
+                env_build.install()
```

Comment:
I would prefer split build and install steps, but this could be made in a next PR. Too much for this one.